### PR TITLE
Profile page skeleton with routing.

### PR DESCRIPTION
**Pull request by sammyswe**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (WilliamMoolman) [inline src/App.css:2]: We can remove this whitespace

File: src/components/Placeholder.jsx
Comment (WilliamMoolman) [inline src/components/Placeholder.jsx:1]: What does placeholder mean? Can we rename this SignIn.jsx?

**Review by WilliamMoolman — CHANGES_REQUESTED** [review]

File: src/components/Placeholder.jsx
Comment (sammyswe) [reply src/components/Placeholder.jsx:1]: Yes on it.

**Review by WilliamMoolman — APPROVED** [review]

**Review by Neutralas — APPROVED** [review]
looks good!

**Pull request merged**